### ISSUE TITLE
Questions about error indicators

### ISSUE THREAD

**Issue by 269448** [Zeying-Gong/PatchMixer#6]
Hello, when I read the code, I found that there was a scale before data input, but there was no anti-standardization when output, which made mae and mse very small. I calculated mape again and reached about 10. mape is not mentioned in the original paper. Is it just the standardization of this operation that makes the non-percentage error like mae seem small? I look forward to your reply and answer

**Comment by Zeying-Gong** [comment]
Thank you for raising these two questions.

1）Instance normalization does indeed include an anti-standardization step. This is handled in PatchMixer.py with the line `x = self.revin_layer(x, 'denorm')`.

2) We primarily compared baselines using metrics such as MSE and MAE. For other objective functions like the MAPE you mentioned, the performance might not be ideal enough. This could require incorporating the loss function for optimization and training.

**Comment by 269448** [comment]
Yes, there is "norm" and" denorm" in your model PatchMixer, which is OK, but there is a scale operation in your data_loader before this, and then it is not transformed back, may I ask why？
